exporter/kinesisexporter/README.md

```diff
@@ -1,3 +1,48 @@
 # Kinesis Exporter
 
-To be added.
\ No newline at end of file
+Kinesis exporter exports Opentelemetry data to Kinesis. This exporter uses a [KPL][kpl-url]-like batch producer and uses
```

Comment:
nitpick: OpenTelemetry

Reply:
nitpick: fill in the right `kpl-url` here.

Reply:
Oh the `kpl-url` is referenced below